### ISSUE TITLE
bug(YouTube - HDR auto brightness): auto HDR brightness does not not work as expected

### ISSUE THREAD

**Issue by 54m4d** [ReVanced/revanced-patches#564]
### Type

Other

### Bug description

The `hdr-auto-brightness` patch/option doesn't seem to have any effect, regardless of whether it's turned on or off.

### Steps to reproduce

1. "Enable brightness gesture" must be on.

2. Enable or disable auto HDR brightness from the video settings, now play an HDR video and swipe up or down to adjust brightness.

3.  Finally notice the difference when auto HDR brightness is turned on and when it's turned off.

### Relevant log output

```shell
Not applicable
```

### Screenshots or videos

![Screenshot_20230722_225246_YouTube](https://github.com/ReVanced/revanced-patches/assets/52612348/3662f913-cac3-4535-b2c7-0168a08af58a)

https://github.com/ReVanced/revanced-patches/assets/52612348/725fe4cb-b102-4df6-a4a3-e5b763e22348

### Additional context

The expected behaviour when the `auto HDR brightness` is off is that the brightness level should follow system brightness, but instead, the brightness increases to high/max when an HDR video is watched in full screen.

Possible relation to ReVanced/revanced-patches-template#1966 

Closing this issue without clarification is not acceptable.

### Acknowledgements

- [X] I have searched the existing issues and this is a new and no duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I filled out all of the requested information in this issue properly.

**Comment by SodaWithoutSparkles** [comment]
> The swipe brightness patch probably needs another hook to turn off or alter the behavior YT is doing for "HDR" videos.  

I dont think so. Swipe brightness controls as-is is already controlling the brightness of a HDR video as expected. 

The brightness cannot be seen from the screen recording, but in-person, the brightness difference is very noticible.

https://github.com/ReVanced/revanced-patches/assets/64138578/4042654c-8397-4e31-be41-d338148ab83f


Currently on patches v4.4.0 with 19.09.37 and manager 1.19.2.

**Comment by SodaWithoutSparkles** [comment]
IMHO, this can be closed as completed.

When swipe brightness set to AUTO, HDR videos follows ambient brightness as expected. This was tested by covering the light sensor.
